package.json

```diff
@@ -3,19 +3,14 @@
   "version": "2.8.0",
   "devDependencies": {
     "babel-core": "~5.8.38",
-    "core-js": "~2.4.0",
-    "custom-event": "~1.0.0",
-    "mocha": "~2.3.2",
-    "phantomjs": "~1.9.18",
     "react": "^16.5.0",
-    "react-dom": "^16.5.0",
-    "simulant": "~0.1.5",
-    "sinon": "~1.16.1",
-    "zuul": "~3.5.0"
+    "react-dom": "^16.5.0"
+  },
+  "peerDependencies": {
+    "react": "^16.5.0",
+    "react-dom": "^16.5.0"
```

Comment:
Is there anything that this package uses that makes this have to be 16.5.0? I think with peer deps, it's good practice to give a range, `> 16.0.0` or use wildcards, like `16.x.x`

Reply:
It's supposed to be 16.5 or greater, which I thought was '^'.

Reply:
I'll take a look at the React changelog and see if I can remember why this uses 16.5.

It will break on any version that doesn't support Portals, but I think those are an older feature.

Reply:
I had a quick look at the commit history and I think we're ok with anything greater than 16.2.
https://github.com/zooniverse/modal-form/commit/02135d88cfaa556aa776f6e51c69b9b1877f3aaf